__tests__/app.test.js

```diff
@@ -34,3 +34,41 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/categories/reviews", () => {
+  test("responds with an array of review objects with the correct properties", () => {
```

Comment:
I would add the status code you're expecting to your test descriptions, by the end of the week you'll have a lot of tests and this addition will make your terminal output much clearer if/when problems occur.